_posts/2019/2019-07-31-consistency.md

```diff
@@ -0,0 +1,108 @@
+
+As already stated in the introduction, it is kind of impossible to have all existing translations in mind.
+Especially if you work on larger or several different projects.
+That's exactly where translation memories jump in.
+They remind you of existing translations even if the source text is not exactly the same.
+By taking the suggestions from the translation memory into account, you can thus easily translate recurring terms and phrases consistently.
+
+## Glossaries
+
+I think we all agree that translation memories are a great way to support consistency in translations.
+But what about consistency in copywriting?
+If there is no text that the translation memory can compare, how should it be able to make suggestions?
+
+So, copywriters have a green field and can come up with whatever they want?
+That's not true either.
+Of course, they also need to take care of using the common terms and phrases.
+But it's a bit harder to find them.
+Glossaries can help here.
+They cannot only be used to explain certain terms and concepts to externals like translation agencies.
+By [maintaining a glossary](https://developer.epages.com/blog/language-and-localization/five-compelling-reasons-why-you-should-care-about-terminology/){:target="_blank"} you also facilitate the life of your copywriters.
+If they are e.g. wondering if they should say *user name* or *username*, or *ecommerce* or *e-commerce*, they can check the spelling in the glossary.
+This way the most important terms of your product will be spelled consistently throughout your software.
+
+## UI text guidelines
+
+Glossaries can support copywriters with the right spelling of a term.
+But what about whole sentences?
+Or the general grammar, and tone of voice?
+You'll need a guideline for these topics.
+The content of the guideline is totally up to you and your product.
+You could start by establishing a structure for recurring text types, and decide how to apply grammar rules to these types.
+This way you can e.g. make sure that all checkbox labels start with a verb.
+
+After [setting up the guideline](https://developer.epages.com/blog/language-and-localization/how-to-create-awesome-ui-texts/){:target="_blank"}, it will simplify your UI text related tasks every single time, and your texts will automatically be more consistent and coherent.
+In the best case, you even set up a guideline for each of your supported locales to give further guidance for your translators. 
+
+## What are you doing that for?
+
+Let me be honest with you.
+Setting up and maintaining all these tools and guidelines is time consuming.
+But it's totally worth it.
+You'll save the time again when creating UI texts, and communicating with your translators, or your developers.
+On top of that you'll save your users time as they can navigate way better and faster through a consistent UI.
+And isn't that enough compensation?
+
+
+------ End of article
+
+## Short summary at the beginning
```

Comment:
Suggestion: 

When it comes to software localization, maintaining consistency is a vital part of the process. In this article we'll share some tips and tricks on how to achieve consistency in software localization in collaboration with developers, copywriters, and translators.